packages/page-template-modal/src/index.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Internal dependencies
+ */
+import './styles/page-template-modal.scss';
```

Comment:
So I don't think we need this import anymore right?

Reply:
yup